0.11 unable to build

Hi! When building 0.11 for Arch Linux I ran into the following:

```
Build BShapr.lv2 DSP...Build BShapr.lv2 GUI...Build BShapr.lv2 (CV version) DSP...Build BShapr.lv2 (CV version) GUI... done.
 done.
 done.
g++: error: BShapr.lv2/tmp/*.o: No such file or directory
make: *** [makefile:175: BShaprGUI-cv.so] Error 1
```

It seems as if although [the directories are seemingly created](https://github.com/sjaehn/BShapr/blob/4106c92bf1d92b7967b2224c227d2279f46a7b93/makefile#L171-L175) (and they exist while compiling!) they are removed before line 175 is executed.

I somehow fixed this by making the `mkdir` calls verbose... I don't quite understand why and I don't think that is the correct solution.
I believe this might be a concurrency issue. Do you usually build this with only one core?

The removal of the build files should ideally be done in a separate `clean` target, as it leads to these rather weird to debug issues during build otherwise.

--- a/makefile
+++ b/makefile
@@ -145,14 +145,13 @@ $(BUNDLE): clean $(DSP_OBJ) $(GUI_OBJ) $(DSP_CV_OBJ) $(GUI_CV_OBJ)
 all: $(BUNDLE)
 
 $(DSP_OBJ): $(DSP_SRC)
-	@echo -n Build $(BUNDLE) DSP...
+	@echo Build $(BUNDLE) DSP...
 	@mkdir -p $(BUNDLE)
 	@$(CXX) $(CPPFLAGS) $(CXXFLAGS) $(LDFLAGS) $(DSPCFLAGS) -Wl,--start-group $(DSPLFLAGS) $< $(DSP_INCL) -Wl,--end-group -o $(BUNDLE)/$@
 	@echo \ done.
 
 $(GUI_OBJ): $(GUI_SRC)
-	@echo -n Build $(BUNDLE) GUI...
-	@mkdir -p $(BUNDLE)
+	@echo Build $(BUNDLE) GUI...
 	@mkdir -p $(BUNDLE)/tmp
 	@cd $(BUNDLE)/tmp; $(CC) $(CPPFLAGS) $(GUIPPFLAGS) $(CFLAGS) $(GUICFLAGS) $(addprefix ../../, $(GUI_C_INCL)) -c
 	@cd $(BUNDLE)/tmp; $(CXX) $(CPPFLAGS) $(GUIPPFLAGS) $(CXXFLAGS) $(GUICFLAGS) $(addprefix ../../, $< $(GUI_CXX_INCL)) -c
@@ -161,23 +160,22 @@ $(GUI_OBJ): $(GUI_SRC)
 	@echo \ done.
 
 $(DSP_CV_OBJ): $(DSP_SRC)
-	@echo -n Build $(BUNDLE) \(CV version\) DSP...
+	@echo Build $(BUNDLE) \(CV version\) DSP...
 	@mkdir -p $(BUNDLE)
 	@$(CXX) $(CPPFLAGS) -DSUPPORTS_CV $(CXXFLAGS) $(LDFLAGS) $(DSPCFLAGS) -Wl,--start-group $(DSPLFLAGS) $< $(DSP_INCL) -Wl,--end-group -o $(BUNDLE)/$@
 	@echo \ done.
 
 $(GUI_CV_OBJ): $(GUI_SRC)
-	@echo -n Build $(BUNDLE) \(CV version\) GUI...
-	@mkdir -p $(BUNDLE)
-	@mkdir -p $(BUNDLE)/tmp
-	@cd $(BUNDLE)/tmp; $(CC) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CFLAGS) $(GUICFLAGS) $(addprefix ../../, $(GUI_C_INCL)) -c
-	@cd $(BUNDLE)/tmp; $(CXX) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CXXFLAGS) $(GUICFLAGS) $(addprefix ../../, $< $(GUI_CXX_INCL)) -c
-	@$(CXX) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CXXFLAGS) $(LDFLAGS) $(GUICFLAGS) -Wl,--start-group $(GUILFLAGS) $(BUNDLE)/tmp/*.o -Wl,--end-group -o $(BUNDLE)/$@
-	@rm -rf $(BUNDLE)/tmp
+	@echo Build $(BUNDLE) \(CV version\) GUI...
+	@mkdir -p $(BUNDLE)/tmp_cv
+	@cd $(BUNDLE)/tmp_cv; $(CC) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CFLAGS) $(GUICFLAGS) $(addprefix ../../, $(GUI_C_INCL)) -c
+	@cd $(BUNDLE)/tmp_cv; $(CXX) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CXXFLAGS) $(GUICFLAGS) $(addprefix ../../, $< $(GUI_CXX_INCL)) -c
+	@$(CXX) $(CPPFLAGS) -DSUPPORTS_CV $(GUIPPFLAGS) $(CXXFLAGS) $(LDFLAGS) $(GUICFLAGS) -Wl,--start-group $(GUILFLAGS) $(BUNDLE)/tmp_cv/*.o -Wl,--end-group -o $(BUNDLE)/$@
+	@rm -rf $(BUNDLE)/tmp_cv
 	@echo \ done.
 
 install:
-	@echo -n Install $(BUNDLE) to $(DESTDIR)$(LV2DIR)...
+	@echo Install $(BUNDLE) to $(DESTDIR)$(LV2DIR)...
 	@$(INSTALL) -d $(DESTDIR)$(LV2DIR)/$(BUNDLE)
 	@$(INSTALL) -d $(DESTDIR)$(LV2DIR)/$(BUNDLE)/inc
 	@$(INSTALL_PROGRAM) -m755 $(B_OBJECTS) $(DESTDIR)$(LV2DIR)/$(BUNDLE)

